src/database/datatools/nomass_materials.py

```
@@ -0,0 +1,146 @@
+        update_description_all_materials(db_path, [am_id, name, 'NoMass', None, material_id])
+
+def delete_nomass_material(db_path: str, nomass_id: int) -> None:
+    conn = sqlite3.connect(db_path)
+    cursor = conn.cursor()
+
+    table_name = "no_mass_materials"
+
+    sql = f"DELETE FROM {table_name} WHERE id = ?"
+    cursor.execute(sql, (nomass_id,))
+
+    sql = f"DELETE FROM all_materials WHERE no_mass_material_id = ?"
```

Ruff (0.14.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)